### PR TITLE
[kube-prometheus-stack] Sync grafana dashboards

### LINKED ISSUE #1022: [kube-prometheus-stack] Grafana Nodes Dashboards no data
**Describe the bug**
Some panels on Nodes dashboards are blank. 
image

**Version of Helm and Kubernetes**:

Helm Version:

```console
$ helm version
Version:"v3.4.1"
```

Kubernetes Version:

```console
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.2", GitCommit:"c97fe5036ef3df2967d086711e6c0c405941e14b", GitTreeState:"clean", BuildDate:"2019-10-15T19:18:23Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.6", GitCommit:"8a62859e515889f07e3e3be6a1080413f17cf2c3", GitTreeState:"clean", BuildDate:"2021-04-15T03:19:55Z", GoVersion:"go1.15.10", Compiler:"gc", Platform:"linux/amd64"}
```

**Which chart**:
kube-prometheus-stack-16.1.1
 APP VERSION 0.48.0

**What you expected to happen**:
I expect to see CPU load  

**How to reproduce it** (as minimally and precisely as possible):
Install this version of chart 

**Changed values of values.yaml** (only put values which differ from the defaults):
```
grafana:
  enabled: true
  namespaceOverride: ""
  ## Deploy default dashboards.
  defaultDashboardsEnabled: true
```

**Anything else we need to know**:

Looks like problem with $__rate_interval. If I change Min step values $__rate_interval -> 1m, everything works fine. 
https://github.com/prometheus-community/helm-charts/commit/cddb167f271812fa7c010019411e59eaa9c1413f#diff-5f2b97c4e53de9ee2be908cbc511915f2d9757ac600058370a3e88ab829fe3c4L97

Also, there is error in browser console
image

### PR DESCRIPTION
**Pull request by kir4h**
Signed-off-by: Andres Alvarez <kir4h.gh@gmail.com>

#### What this PR does / why we need it

This PR syncs grafana dashboards. It resolves current issue presenting some metrics like CPU empty because of min step set at `$__rate_interval` (see #1022) 

#### Which issue this PR fixes
  - fixes #1022

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by Xtigyro — CHANGES_REQUESTED** [review]
Thanks for the contribution, @kir4h! 🥇 

Could you please work together with @timbrd - so that we can get his changes (https://github.com/prometheus-community/helm-charts/pull/1044) not overwritten with this PR?

**Comment by kir4h** [comment]
@Xtigyro I will take a look at the sync_grafana_dashboards to avoid overwritting the changes

**Comment by Xtigyro** [comment]
@kir4h Thank you!

File: charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/apiserver.yaml [hunk 1]
```diff
@@ -1744,4 +1744,4 @@ data:
         "uid": "09ec8aa1e996d6ffcd6817bbaff4db1b",
         "version": 0
     }
-{{- end }}
+{{- end }}
```
Comment (Xtigyro) [inline charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/apiserver.yaml:1747]: @kir4h Could you please retain the empty line at the end of the many files you've edited?

**Review by Xtigyro — CHANGES_REQUESTED** [review]

File: charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/apiserver.yaml (hunk 1 above)
Comment (kir4h) [reply charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/apiserver.yaml:1747]: If I'm not mistaken, by doing so I would "unsync" the script again (I have tested regenerating the dashboards from the CLI (to ensure no IDE is removing them) and there are no changes in git), as the empty line is not automatically generated.
It seems it was manually introduced in https://github.com/prometheus-community/helm-charts/pull/1044  (example: https://github.com/prometheus-community/helm-charts/pull/1044/files#diff-80b84898a52648f05e7a7caf69cf86be7fa57179f2ec9bc28f54d6e297a65305R1747)

Comment (Xtigyro) [reply charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/apiserver.yaml:1747]: @kir4h You're right!

**Comment by kir4h** [comment]
@Xtigyro out of curiosity: why are we bumping major version here? 

When I first created this PR I bumped the patch version as it was a fix on a dashboard. As a side effect, other dashboard was updated changing some logic in persistentvolumes metrics, so that could be the reason for a transitive minor. But can't figure out why bumping the major now.

Thanks!

**Comment by Xtigyro** [comment]
@kir4h Making any backwards incompatible changes should lead to a major version bump. I guess we can always argue whether that's exactly the case but better be on the safe side.

https://semver.org/

**Comment by Xtigyro** [comment]
@kir4h I've checked some of the previous PR's when similar stuff has been done. Most people lean towards bumping just the minor version. So let's do that here as well.

**Review by Xtigyro — APPROVED** [review]
@kir4h Thank you!

**Pull request merged**